Monika After Story/game/script-topics.rpy

```diff
@@ -6937,6 +6937,306 @@ label monika_citizenship:
+                m 2esa "Sure, [player]."
+                m 2eka "I hope whatever happened last time wasn't too serious and that things have calmed down now."
+                m "It really means a lot to me that you want to hear my speech again after you weren't able to listen to the whole thing before."
+                m 2hua "Okay, I'll get started now!"
+
+            else:
+                m 2ekc "Okay, [player], but I hope you actually listen this time."
+                m 2dkd "It really hurt me when you didn't pay attention."
+                m 2dkc "..."
+                m 2eka "I do appreciate you asking to hear it again, so I'll get started now."
+
+            #say speech
+            call monika_speech
+
+            #another timed menu checking if you were listening
+            show screen mas_background_timed_jump(10, "monika_ignored_lock") 
+            menu:
+                m "So, [player], now that you actually {i}heard{/i} my speech, what do you think?"
+                #If menu is used, set player on a good path
+                "It's great! I'm so proud of you!":
+                    hide screen mas_background_timed_jump
+                    $ mas_gainAffection(amount=3, bypass=True)
+                    $ persistent._mas_monika_listened = True
+
+                    m 2subfb "Aww, [player]!"
+                    m 2ekbfa "Thank you so much! I worked really hard on that speech, and it means so much to me that you gave it another chance."
+                    m "Hearing that you're proud of me as well makes it that much better."
+                    show monika 5eubfu at t11 zorder MAS_MONIKA_Z with dissolve
+                    m 5eubfu "As much as I wish I could have given my speech in front of everyone, just having you by my side is so much better."
+                    m 5eubfb "I love you, [player]!"
+
+                "I like it!":
+                    hide screen mas_background_timed_jump
+                    $mas_gainAffection(amount=1, bypass=True)
+                    $ persistent._mas_monika_listened = True
+
+                    m 2eka "Thanks for listening this time, [player]..."
+                    m "I'm so glad you enjoyed it."                   
+
+                "That {i}was{/i} long":
+                    hide screen mas_background_timed_jump
+                    $mas_loseAffection(modifier=2)
+                    $ persistent._mas_monika_listened = True
+
+                    m 2tfc "After acting like you actually wanted me to recite it for you again, {i}that's{/i} what you have to say?"
+                    m 2dfc "..."
+                    m 6lktdc "I really thought after I told you how important this was to me, {i}{w=1}twice{/i}, {w=1}you would have been more supportive and let me have my moment."
+                    m 6ektdc "All I wanted was for you to be proud of me, [player]..."
+                    m 6dstsc "But I guess that's too much to ask."
+    return
+
+label monika_not_paying_attention:
```

Comment:
label names should include the parent name so we can keep track of what belongs to what. change this to something like
`monika_grad_speech_not_paying_attention`